ansible/roles/test/files/ptftests/dhcp_relay_test.py

```diff
@@ -139,6 +145,12 @@ def setUp(self):
         self.option82 += struct.pack('BB', 2, len(remote_id_string))
         self.option82 += remote_id_string
 
+        # In 'dual' testing mode, vlan ip is stored as suboption 5 of option 82.
```

Comment:
Suggest adding a breakdown of the bytes in suboption 5 as I have done above so that the code is more understandable.

Reply:
updated, thanks

Reply:
updated, thanks